semantics/llvm-syntax.k

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013-2014 K Team. All Rights Reserved.
+/*@
+
+This is the syntax of LLVM.
+
+This syntax requires a input LLVM program without any comments on it.
```

Comment:
It is hard to define LLVM comments in the current K framework because the comment separated character is "//" in K; however, the comment separated character is ";" in LLVM. In order to define such comments, we need to define ";" in every possible place it appears in a LLVM program. I think we discussed this issue before and you claimed that we should not worry about this now.